redfish/types.py

```diff
@@ -451,7 +470,8 @@ def __init__(self, url, connection_parameters):
 class EthernetInterfacesCollection(BaseCollection):
     '''Class to manage redfish EthernetInterfacesColkection data.'''
     def __init__(self, url, connection_parameters):
-        super(EthernetInterfacesCollection, self).__init__(url, connection_parameters)
+        super(EthernetInterfacesCollection,
```

Comment:
Yes, this is to respect 80 chars per line. I agree in that case not respecting PEP make it clearer.
However I have plan some changes around connection_parameters so this line will be probably shorter at the end.